services-api/src/main/java/io/scalecube/services/IMicroservices.java

```diff
@@ -0,0 +1,13 @@
+package io.scalecube.services;
+
+import io.scalecube.net.Address;
+import io.scalecube.services.discovery.api.ServiceDiscovery;
+
+public interface IMicroservices {
```

Comment:
how about to rename current microservices class to ScaleCube

so:

`ScaleCube implements Microservices`